calabash/Classes/FranklyServer/Routes/LPBackdoorRoute.m

```diff
@@ -29,40 +34,41 @@ - (NSDictionary *) JSONResponseForMethod:(NSString *) method URI:(NSString *) pa
     selectorName = [selectorName stringByAppendingString:@":"];
   }
 
+  id argument = [data objectForKey:@"arg"];
+  if (!argument) {
+    LPLogError(@"Expected data dictionary to contain an 'arg' key");
```

Comment:
Is it really a problem is the user doesn't supply an argument? Objective-C was designed to be robust to `nil` pointer operations, and it's acceptable in general to pass `nil` as a method param. Plus, idiomatically, it's odd to write methods like:

`- (void)changeAppState:(id)ignoreMe { /**/ } `